controllers/argocd/util.go

```diff
@@ -1477,6 +1489,43 @@ func (r *ReconcileArgoCD) cleanupUnmanagedSourceNamespaceResources(cr *argoproj.
 	return nil
 }
 
+func (r *ReconcileArgoCD) cleanupAllSourceNamespaces(cr *argoproj.ArgoCD) {
```

Comment:
Do we need `cleanupAllSourceNamespaces`? It seems like it's doing the exact same thing as `removeUnmanagedSourceNamespaceResources`?